src/dynamo0p3.py

```diff
@@ -1569,17 +1794,28 @@ def gen_code(self, parent):
                 TypeDeclGen(invoke_sub,
                             datatype="operator_proxy_type",
                             entity_decls=op_proxy_decs))
+        cma_op_proxy_decs = self.unique_proxy_declarations(
+            "gh_columnwise_operator")
+        if len(cma_op_proxy_decs) > 0:
```

Comment:
I'm not sure that was deliberate and your suggestion is more pythonic so I've changed my code (and resisted the temptation to change the others).